src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,14 @@
+<Configuration status="INFO">
```

Comment:
Can we better move this file to `src/test/resources`? Would be better to avoid packaging it into our localstack-utils JAR.